bayesian_cnn_prometheus/preprocessing/healthy_patients_indices.json

```diff
@@ -0,0 +1 @@
+{"healthy_patients_indices": ["0002", "0003", "0001", "0004", "0005"]}
```

Comment:
Można by było jeszcze bardziej oszczędzić pamięć, i zapisywać same int'y, a potem przy wczytywaniu robić padding zerami za pomocą [f-stringów](https://stackoverflow.com/questions/339007/how-to-pad-zeroes-to-a-string)
:D